internal/config/backup_test.go

```diff
@@ -48,44 +49,87 @@ func TestBackupManager_Bind(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       fields: fields {
-		           Host: "",
-		           Port: 0,
-		           Client: GRPCClient{},
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           fields: fields {
-		           Host: "",
-		           Port: 0,
-		           Client: GRPCClient{},
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "return Backup when client is nil",
+			fields: fields{
+				Client: nil,
+			},
+			want: want{
+				want: &BackupManager{
+					Client: newGRPCClientConfig(),
```

Comment:
could you please define GRPCClient not using `newGRPCClientConfig()`?

Reply:
fixed